src/Server.php

```diff
@@ -90,6 +90,25 @@ public function startBrokerSession()
 
         $this->brokerId = $this->validateBrokerSessionId($sid);
     }
+		
+	/**
+	 * Get session ID from header Authorization or from $_GET/$_POST
+	 */
+	protected function getBrokerSessionID(){
+		$headers = getallheaders();
+		
+		if (isset($headers['Authorization'])){
+			return $headers['Authorization'];
+		}
+		if (isset($_GET['sso_session'])) {
+			return $_GET['sso_session'];
+		}
+		if (isset($_POST['sso_session'])) {
```

Comment:
Please follow RFC 6750 and accept `$_GET['access_token']` or `$_POST['access_token']`. Also accept `$_GET['sso_session']` for backwards compatibility.